### PR TITLE
schema and settings changes to meet various customer needs

### PR DESCRIPTION
**Pull request by jimduff-usds**
This PR ...
- adds patAgeUnits to HHSProtect.
- adds child of 'direct' schema, for AnavasiDx.   Added it to orgs.yml as well.
- add row for Proctored-eligible test into the LIVD-Supplemental table.
- fixed spelling of 'Johns'. ;) 
- Added CA secondary feed in orgs.yml
- Filter for HHSProtect in PROD to now allow Waters data to flow to HHSProtect

### Testing
- [x] Tested locally?
- [x] Ran `quickTest all`?
- [x] Ran `./prime test` against local Docker ReportStream container?
- [ ] Downloaded a file from `http://localhost:7071/api/download`?
- [ ] Added tests?

### Specific Security-related subjects a reviewer should pay specific attention to

- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [ ] Includes a summary of what a code reviewer should verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

### CODE REVIEW

File: prime-router/settings/prod/0034-hhsprotect.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+topic: "covid-19"
+translation: !<CUSTOM>
+  schemaName: "hhsprotect/hhsprotect-covid-19"
+  format: "CSV"
+  defaults: {}
+  nameFormat: "STANDARD"
+  receivingOrganization: null
+  type: "CUSTOM"
+jurisdictionalFilter:
+- "matches(sender_id, SafeHealth,ImageMover,Cue,InBios,Strac,.*CueHlth.*,.*ImageMover.*)"
```
Comment (jimduff-usds) [inline prime-router/settings/prod/0034-hhsprotect.yml:16]: `.*CueHlth.*,.*ImageMover.*` means that any data with sender_id that contains `CueHlth` or `ImageMover` will now go to HHSProtect.  This should include almost all data coming from Waters.   (SafeHealth is already dual sending)

File: prime-router/settings/prod/0035-ihs-az-sender.yml [hunk 2]
```diff
@@ -1,10 +1,10 @@
 # Run this:   ./prime multiple-settings set --input ./settings/prod/0035-waters.yml --env prod
```
Comment (jimduff-usds) [inline prime-router/settings/prod/0035-ihs-az-sender.yml:1]: Just spelling fix here.

File: prime-router/settings/prod/0034-hhsprotect.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Run ./prime receiver set --input ./settings/prod/0030-hhsprotect.yml --env prod
```
Comment (jimduff-usds) [inline prime-router/settings/prod/0034-hhsprotect.yml:1]: Note for this and the next few files:

Any file `prime-router/settings/prod/NNNN-xxx.yml` and/or `prime-router/settings/staging/MMMM-yyy.yml ` are "yamlitos" - little snippets of receiver and/or sender configuration changes/additions/subtractions destined for prod or staging respectively.

These changes are not deployed - they ONLY take effect after the file has been submitted to the corresponding prod or staging API that handles settings changes.  Currently, this is done manually.

So, the yamlitos are only here as a convenience to be reviewed; they are not part of the build.

File: prime-router/src/main/kotlin/azure/ReportFunction.kt [hunk 4]
```diff
@@ -490,15 +490,18 @@ class ReportFunction {
             // the report has all the submitted items
             report.itemIndices.forEach { i ->
                 // if an item was not present, create the routing with empty destinations
-                items.add(routingMap.getOrDefault(
```
Comment (jimduff-usds) [inline prime-router/src/main/kotlin/azure/ReportFunction.kt:493]: fixing ktlint issues

File: prime-router/metadata/tables/LIVD-Supplemental-2021-06-07.csv [hunk 5]
```diff
@@ -8,4 +8,5 @@ QuickVue At-Home OTC COVID-19 Test*,95209-3,QuickVue At-Home OTC COVID-19 Test_Q
...
 Ellume COVID-19 Home Test_Ellume Limited*,95209-3,Ellume COVID-19 Home Test_Ellume Limited,,Y,Y,N,Y,https://www.fda.gov/media/144457/download,"The Ellume COVID-19 Home Test is authorized for non-prescription home use"
 Cue COVID-19 Test for Home and Over The Counter (OTC) Use,95409-9,Cue COVID-19 Test for Home and Over The Counter (OTC) Use_Cue Health Inc.,,Y,Y,N,Y,https://www.fda.gov/media/146467/download,"The Cue COVID-19 Test for Home and Over The Counter (OTC) Use is authorized for non-prescription home use"
+Cue COVID-19 Test for Home and Over The Counter (OTC) Use (Proctored),95409-9,Cue COVID-19 Test for Home and Over The Counter (OTC) Use (Proctored)_Cue Health Inc.,,Y,Y,N,N,https://www.fda.gov/media/146467/download,"The Cue COVID-19 Test for Home and Over The Counter (OTC) Use is authorized for non-prescription home use"
```
[the remark this answers is not in the document]
Comment (carlosfelix2) [reply prime-router/metadata/tables/LIVD-Supplemental-2021-06-07.csv:11]: Why is this a temporary hack?  Why temporary?

File: prime-router/settings/organizations.yml [hunk 6]
```diff
@@ -97,12 +107,22 @@
       schemaName: direct/careevolution-covid-19
       format: HL7
 
-- name: "john_hopkins"
-  description: "John Hopkin's / IHS school testing program"
+- name: primary
```
Comment (carlosfelix2) [inline prime-router/settings/organizations.yml:110]: Primary is such a generic word.  Maybe primaryDiag?  Not sure what naming convention has been used.

**Review by carlosfelix2 — APPROVED** [review]

File: prime-router/settings/organizations.yml (hunk 6 above)
Comment (jimduff-usds) [reply prime-router/settings/organizations.yml:110]: Good point.   my informal convention has been to use the shorthand name that companies use to refer to themselves

File: prime-router/metadata/tables/LIVD-Supplemental-2021-06-07.csv (hunk 5 above)
[the remark this answers is not in the document]
Comment (jimduff-usds) [reply prime-router/metadata/tables/LIVD-Supplemental-2021-06-07.csv:11]: its a hack to list a device that does not exist in the official LIVD table.    Its really a flaw in our underlying design - it turns out this customer can have both a non-proctored and proctored test, but still using the same medical device info in the LIVD table.   Our design assumes these would get registered as two separate medical devices, and so the deviceID could be used to differentiate whether a test is proctored or not.

**Pull request merged**